_docs/info/migrate_rpool.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Migrate rpool to new disk(s)
+category: info
+show_in_sidebar: true
+---
+
+## Migrate rpool to new disk(s)
+
+Migrating the rpool is not just a matter of zfs send and zfs receive since the name of the pool rpool is used literally various places in the core configuration so the name must not be changed. Apart from this the disk(s) containing the rpool also contains the boot block and the boot loader which is not part of the pool and therefore must be dealt with using other tools than zfs. therefore this short guide will give instructions for the simplest and easiest way to accomplish this task.
+
+This guide is using a mirrored rpool as an example so if your own rpool just consists of one disk you can use this guide too just using one disk where the guide talks of two disks.
+
+NB. **It is always a good idea to perform a scrub before doing operations on the pool which is potentially dangerous!.**
+
+## Configuration used in this guide
+
+This is the current rpool:
+
+``` terminal
+$ zpool status
+  pool: rpool
+ state: ONLINE
+  scan: scrub repaired 0 in 3h6m with 0 errors on Sat Apr  11 23:21:26 2020
+config:
+
+        NAME        STATE     READ WRITE CKSUM
+        rpool       ONLINE       0     0     0
+          mirror-0  ONLINE       0     0     0
+            c1t1d0  ONLINE       0     0     0
+            c1t2d0  ONLINE       0     0     0
+
+```
+
+Disks available in the system:
+
+```terminal
+AVAILABLE DISK SELECTIONS:
+       0. c1t1d0 <VBOX-HARDDISK-1.0-16.00GB>
+          /pci@0,0/pci8086,2829@1f,2/disk@1,0
+       1. c1t2d0 <VBOX-HARDDISK-1.0-16.00GB>
+          /pci@0,0/pci8086,2829@1f,2/disk@2,0
+       2. c1t3d0 <VBOX-HARDDISK-1.0-16.00GB>
+          /pci@0,0/pci8086,2829@1f,2/disk@3,0
+       3. c1t4d0 <VBOX-HARDDISK-1.0-16.00GB>
+          /pci@0,0/pci8086,2829@1f,2/disk@4,0
+
+```
+
+The task is to migrate the rpool from the mirror consisting of c1t1d0 and c1t2d0 to a new mirror consisting of c1t3d0 and c1t4d0.
+
+So after completing this guide your rpool should look like this:
+
+``` terminal
+$ zpool status
+  pool: rpool
+ state: ONLINE
+  scan: resilvered 1.59G in 0h11m with 0 errors on Sun Apr 12 00:35:53 2020
+config:
+
+        NAME        STATE     READ WRITE CKSUM
+        rpool       ONLINE       0     0     0
+          mirror-0  ONLINE       0     0     0
+            c1t3d0  ONLINE       0     0     0
+            c1t4d0  ONLINE       0     0     0
+
+```
+
+## Attach the new disk(s) to rpool
+
+The first step is to attach the new disks to the existing rpool using the command **zpool attach** like this:
+
+``` terminal
+$ sudo zpool attach rpool c1t1d0 c1t3d0
+Password: 
+Make sure to wait until resilver is done before rebooting.
+```
+
```

Comment:
Could you add the `zpool status` output here so people can see what this attach command did to the pool layout?